doc/pages/upgrade/030-upgrade_100_101.md

```diff
@@ -1,9 +1,9 @@
-# Upgrade v1.0.0-v1.0.1
+# Upgrade v1.0.0-v1.0.1 (or later)
```

Comment:
Instead of using "or later" I'd suggest using `Upgrade v1.0.0-v1.0.x`. This will make clear that the upgrade path is not valid for a minor version change, i.e. 1.1.